pkg/sql/sqlbase/structured.go

```diff
@@ -212,6 +212,10 @@ type ImmutableTableDescriptor struct {
 	// progress, as mutation columns may have NULL values.
 	ReadableColumns []ColumnDescriptor
 
+	// columnsWithUDTs is a set of indexes into publicAndNonPublicCols containing
+	// indexes of columns that contain user defined types.
+	columnsWithUDTs []int
```

Comment:
This could be a boolean, but I forsee that it will be useful if we remember what columns actually are user defined here.